internal/namespaces/init/init.go

```diff
@@ -65,6 +66,7 @@ type initArgs struct {
 	SendTelemetry       *bool
 	WithSSHKey          *bool
 	InstallAutocomplete *bool
+	V1ConfigFilePath    *string
```

Comment:
This is not an argument of the command.
If we really want to make it an arg we could do
`scw init delete-v1-config=true`
but i'm not a fan